tests/test_stratis.py

```
@@ -137,30 +137,30 @@ def test__get_pool_filesystem(self):
-        self.assertTrue(str(pool.Name()) == "pool1")
+        (pool, filesystem) = stratis._get_pool_filesystem(managed_objects, _POOL_NAME, "fs1")
+        self.assertTrue(str(pool.Name()) == _POOL_NAME)
         self.assertTrue(str(filesystem.Name()) == "fs1")
 
     def test__get_pool_filesystem_bad_pool(self):
         proxy = get_object(TOP_OBJECT)
         managed_objects = ObjectManager.Methods.GetManagedObjects(proxy, {})
 
         with self.assertRaises(DbusClientUniqueResultError) as cm:
-            (pool, filesystem) = stratis._get_pool_filesystem(managed_objects, "nosuchpool1", "fs1")
+            (pool, filesystem) = stratis._get_pool_filesystem(managed_objects, "nosuchpool", "fs1")
```

Ruff (0.15.21)

[warning] Unpacked variable `pool` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)